semconv/v1.4.0/http.go

```diff
@@ -225,7 +225,9 @@ func HTTPServerAttributesFromHTTPRequest(serverName, route string, request *http
 		attrs = append(attrs, HTTPRouteKey.String(route))
 	}
 	if values, ok := request.Header["X-Forwarded-For"]; ok && len(values) > 0 {
-		attrs = append(attrs, HTTPClientIPKey.String(values[0]))
+		if addresses := strings.Split(values[0], ","); len(addresses) > 0 {
+			attrs = append(attrs, HTTPClientIPKey.String(addresses[0]))
+		}
```

Comment:
So the request.Headers is a map of []string.  Are you sure that the HTTP library doesn't already do this split for us?

Reply:
According to the examples in https://pkg.go.dev/net/http#Request, the Go `Header` type will already be split as @MadVikingGod says. The docs aren't very convincing that this will always be the case, so this change looks defensible.